tools/batchTestGenerator/internal/config.go

```diff
@@ -56,8 +56,22 @@ type ClusterTarget struct {
 }
 
 type Target struct {
-	Name   string `json:"name"`
-	Region string `json:"region"`
+	Name            string   `json:"name"`
+	Region          string   `json:"region"`
+	ExcludedTests   []string `json:"excluded_tests"`
+	excludedTestMap map[string]struct{}
+}
+
+func (t *Target) isTestCaseExcluded(value string) bool {
+	if t.excludedTestMap == nil {
```

Comment:
if `excludedTestMap == nil` why not return `false` right away instead of populating this map with empty data?

Reply:
I see why now after reading @Aneurysm9 's comment.